packages/meditrak-server/src/tests/routes/accessRequests/EditAccessRequests.test.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { expect } from 'chai';
+import { findOrCreateDummyRecord, findOrCreateDummyCountryEntity } from '@tupaia/database';
+import { Authenticator } from '@tupaia/auth';
+import {
+  TUPAIA_ADMIN_PANEL_PERMISSION_GROUP,
+  BES_ADMIN_PERMISSION_GROUP,
+} from '../../../permissions';
+import { TestableApp } from '../../TestableApp';
+import { prepareStubAndAuthenticate } from '../utilities/prepareStubAndAuthenticate';
+
+describe('Permissions checker for EditAccessRequests', async () => {
+  const DEFAULT_POLICY = {
+    DL: ['Public'],
+    KI: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Admin'],
+    SB: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Royal Australasian College of Surgeons'],
+    VU: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Admin'],
+    LA: ['Admin'],
+    TO: ['Admin'],
+  };
+
+  const BES_ADMIN_POLICY = {
+    LA: [BES_ADMIN_PERMISSION_GROUP],
+  };
+
+  const app = new TestableApp();
+  const { models } = app;
+  let vanuatuPublicRequest;
+  let laosPublicRequest;
+  let kiribatiBESRequest;
+  let laosEntityId;
+  let BESPermissionGroupId;
+
+  before(async () => {
+    const { entity: vanuatuEntity } = await findOrCreateDummyCountryEntity(models, {
+      code: 'VU',
+    });
+
+    const { entity: kiribatiEntity } = await findOrCreateDummyCountryEntity(models, {
+      code: 'KI',
+    });
+    const { entity: laosEntity } = await findOrCreateDummyCountryEntity(models, {
+      code: 'LA',
+    });
+    laosEntityId = laosEntity.id;
+
+    const publicPermissionGroup = await findOrCreateDummyRecord(models.permissionGroup, {
+      name: 'Public',
+    });
+    const BESPermissionGroup = await findOrCreateDummyRecord(models.permissionGroup, {
```

Comment:
```suggestion
    const besPermissionGroup = await findOrCreateDummyRecord(models.permissionGroup, {
```
```suggestion
    const permissionGroupBES = await findOrCreateDummyRecord(models.permissionGroup, {
```
The capital first letter makes it feel like a class name...